simphony/cuds/mesh.py

```diff
@@ -676,6 +684,33 @@ def has_cells(self):
         """
         return len(self._cells) > 0
 
+    def count_of(self, item_type):
+        """ Return the count of item_type in the container.
+
+        Parameter
+        ---------
+        item_type : CUBA
+            The CUBA enum of the type of the items to return the count of.
+
+        Returns
+        -------
+        count : int
+            The number of items of item_type in the container.
+
+        Raises
+        ------
+        ValueError :
+            If the type of the item is not supported in the current
+            container.
+
+        """
+
+        try:
+            return len(self._items_count[item_type]())
+        except:
```

Comment:
In anycase if you fill that capturing KeyError is fragile (which sounds a little extreme) then please use an `if...else`. In anycase a bare `except` is not right.